### ISSUE TITLE
Cannot delete non-empty directory

### ISSUE THREAD

**Issue by dhruvmanila** [tamago324/lir.nvim#24]
The function used to delete `vim.loop.fs_rmdir` which is similar to `rmdir` utility cannot delete directories which are non-empty:
```lua
{
  [2] = "ENOTEMPTY: directory not empty: /Users/dhruvmanilawala/dotfiles/config/nvim/scratch/test",
  [3] = "ENOTEMPTY"
}
```

The confirmation prompt asks for the `force` option as well but that does not seem to be doing anything nor used in the code.

**Comment by tamago324** [comment]
Thanks for the report.
I fixed 84979aa65fbedb58d4d3c275b82c89109e2e0167.
May I ask for confirmation?

**Comment by dhruvmanila** [comment]
Yes, it works, thanks for fixing it :)
